src/main/resources/policies/document-rechten.rego

```diff
@@ -73,11 +73,12 @@ verwijderen {
     zaaktype_allowed
     document.zaak_open == true
     document.definitief == false
-    onvergrendeld_of_vergrendeld_door_user == true
+    document.vergrendeld == false
```

Comment:
Just checking: our policy matrix (https://github.com/infonl/dimpact-zaakafhandelcomponent/blob/main/docs/solution-architecture/accessControlPolicies.md) does not need to be changed because of this?

Reply:
Good call, it does indeed!

Reply:
Done

Reply:
Follow-up question: this change (in the matrix) is according to the user story or as agreed with the PO? I guess so. :-)

Reply:
I'll ask explicitly and report back here